teku/src/main/java/tech/pegasys/teku/cli/deposit/RegisterAction.java

```diff
@@ -100,8 +110,10 @@ public void displayConfirmation(final int totalNumberOfDeposits) {
         consoleAdapter.readLine(
             "You are about to submit "
                 + transactionPart
-                + " of %s Eth. This is irreversible, please make sure you understand the consequences. Are you sure you want to continue? [y/n]",
-            eth);
+                + " of %s Eth to contract address [%s] on network [%s].\nThis is irreversible, please make sure you understand the consequences. Are you sure you want to continue? [y/n]",
```

Comment:
This is misleading.  The transfer is happening on the ETH1 network but this statement is reporting the ETH2 network.  What network the transfer happens on is controlled by what chain the ETH1 endpoint is sync'd to and nothing this tool can do will change that.

Reply:
Maybe it could say something like "The deposit signature will be valid for the Ethereum 2 network %s"?  Although the other catch is that the ETH2 network may be a URL to a YAML file with the network config, it's not necessarily a nice human friendly name.